test/functional/editor_controller_test.rb

```diff
@@ -45,8 +45,10 @@ def setup
         tags: 'question:question,one',
         template: 'question',
         redirect: 'question'
-    assert_select "h3", "Ask a question of the community"
-    assert_select "input#taginput[value=?]", "question:question,one"
+        assert_response :redirect
```

Comment:
Sorry to worry about details, but if you could adjust this indent (two spaces per indent) that'd be great. It's helpful when the whole codebase has really standardized formatting. Thanks!